daemons/lvmdbusd/lv.py

```diff
@@ -232,7 +232,6 @@ def creation_signature(self):
 @utils.dbus_property(LV_COMMON_INTERFACE, 'Attr', 's')
 @utils.dbus_property(LV_COMMON_INTERFACE, 'DataPercent', 'u')
 @utils.dbus_property(LV_COMMON_INTERFACE, 'SnapPercent', 'u')
-@utils.dbus_property(LV_COMMON_INTERFACE, 'DataPercent', 'u')
```

Comment:
The third parameter is the dbus type specifier `u` is a **Unsigned 32-bit integer**.  This is part of the public API specification, so right or wrong it's part of the API.  We can certainly add new properties which are a double, but we have clients which now expect that this is an unsigned 32 bit integer.

Reply:
Yes I know, possibly its a bad idea to break the API :smile:. But the line your referencing is a duplicate and could be removed.

Reply:
Like I mentioned, we can certainly fix bugs, and having a duplicate line here is indeed a bug.

Reply:
I'll rework the necessary code, so you possibly just need squash on merge.